newsfragments/42360.significant.rst

```diff
@@ -0,0 +1,13 @@
+``uri`` is replaced with ``id`` in ``AssetDetails`` . Hence, ``is_authorized_asset`` method needs to be updated in Auth Managers to use ``id`` instead of ``uri``.
```

Comment:
@vincbeck , @pierrejeambrun , I've added this newsfragment after referring to other newsfragments. does this look good or do I need to change this?

Reply:
Yes, it should be the PR number. Other than that, wording looks good